modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -100,7 +127,7 @@ curl --json '{"model_id": "<model_name>", "inputs": "<text>"}' https://<inferenc
 *TGIS Standalone ServingRuntime for KServe*
 [source]
 ----
-grpcurl -proto text-generation-inference/proto/generation.proto -d '{"requests": [{"text":"<text>"}]}' -H 'mm-model-id: <model_name>' -H 'Authorization: Bearer <token>' -insecure <inference_endpoint_url>:443 fmaas.GenerationService/Generate  <1>
+grpcurl -proto text-generation-inference/proto/generation.proto -d '{"requests": [{"text":"<text>"}]}' -H 'Authorization: Bearer <token>' -insecure <inference_endpoint_url>:443 fmaas.GenerationService/Generate  <1>
```

Comment:
Same as previous question - is it no longer necessary to specify the model name for the TGIS runtime?

Reply:
It actually has never been necessary as far as I know, not sure how it made into the docs, but since it has a `mm-` prefix, I;m assuming it's some modelmesh example that got copied over and wasn't cleaned of unneeded args.